README.md

```diff
@@ -270,7 +277,7 @@ Text('money').plural(1000000, format: NumberFormat.compact(locale: context.local
 print('day'.plural(21)); // output: 21 день
 
 //Static function
-var money = plural('money', 10.23) // output: You have 10.23 dollars
```

Comment:
don't need delete this, just stay two variants 
```
//Static function
var money = plural('money', 10.23) // output: You have 10.23 dollars

//Static function with arguments
var money = plural('money_args', 10.23, args: ['John', '10.23'])  // output: John has 10.23 dollars
```

Reply:
@Overman775 got it. I just update it. Please check it out. Thanks :)